src/wmtk/PolygonMesh.cpp

```diff
@@ -0,0 +1,239 @@
+#include "PolygonMesh.hpp"
+
+namespace wmtk {
+PolygonMesh::PolygonMesh()
+    : Mesh(2)
+    , m_en_handle(register_attribute<long>("m_en", PrimitiveType::Edge, 2))
+    , m_ep_handle(register_attribute<long>("m_ep", PrimitiveType::Edge, 2))
+    , m_ev_handle(register_attribute<long>("m_ev", PrimitiveType::Edge, 2))
+    , m_ef_handle(register_attribute<long>("m_ef", PrimitiveType::Edge, 2))
+    , m_fh_handle(register_attribute<long>("m_fh", PrimitiveType::Face, 1))
+    , m_vh_handle(register_attribute<long>("m_vh", PrimitiveType::Vertex, 1))
+    , m_f_is_hole_handle(register_attribute<bool>("m_f_is_hole", PrimitiveType::Face, 1))
```

Comment:
i don't think we support bool - just char right?

TBH this is one thing where i suspect teh flagaccessor could be used (first bit is dedicated to whether a cell is active or inactive, the next bit could be used for this purpose